Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -51,6 +51,8 @@ struct _WPEViewPrivate {
     gdouble scale { 1 };
     WPEViewState state;
     bool closed;
+    bool visible { true };
+    bool mapped;
```

Comment:
Shouldn't we default to false here? (ah c struct :-)

Reply:
The view is visible by default